### DISCUSSION TITLE
Upgrade v15 from 14 failed! Migration service may be blocked in some countries.

### DISCUSSION THREAD

**Discussion by wendajiang** [blinksh/blink#1522] [Support] [answered]
I'm a Chinese user, and when I upgrade v15 from v14,  in the Step 1 Purchase validation: validating receipt, it's always tips: Error performing your migration request.

If the request to the fixed domain, and I use the VPN can access google.com, but I do not know the request domain to set proxy to request.

**Comment by carloscabanero** [comment, marked as answer]
Hi @wendajiang !! The issue is that the app cannot contact our service to validate the receipt. This is because in some countries, some internet providers are blocking Google Functions, which is where we are hosting our service. Could you please give it a try under a different network? Some users have success when doing it through cellular, or using a different VPN, etc... The domain is on https://us-central1-gold-stone-332203.cloudfunctions.net.

Let us know if that helped!

**Reply by wendajiang** [reply]
Thanks! I can set the rule (doamin-suffix) use my VPN. And It's worked.
